Gemfile

```diff
@@ -32,9 +32,15 @@ gem 'jbuilder'
 # user authentication
 gem 'devise'
 
+# user authorization
+gem 'action_policy'
```

Comment:
TODO: action policy is kind of annoying, maybe consider ripping it out and replacing it with a homegrown authorization system. It doesn't need to be very robust